packages/venia-concept/package.json

```diff
@@ -23,8 +23,7 @@
     "prettier:fix": "yarn run -s prettier -- --write",
     "start": "node server.js",
     "start:debug": "node --inspect-brk ./node_modules/.bin/webpack-dev-server --progress --color --env.mode development",
-    "storybook": "echo 'Venia component stories have moved to @magento/venia-ui. Trying to run in sibling directory...' && (cd ../venia-ui && yarn run storybook:build)",
-    "storybook:build": "yarn run storybook",
+    "storybook": "start-storybook -p 9001 -c src/.storybook",
```

Comment:
Can you add also a `storybook:build` to get something that we can deploy somewhere?